mongodb_exporter.go

```diff
@@ -239,6 +263,18 @@ func main() {
 	}
 	flag.Parse()
 
+	uri := os.Getenv("MONGODB_URI")
+	if uri != "" {
```

Comment:
btw, should we check that `uriF` is empty?

Reply:
Hmm, why, I don't understand? I understand that if `MONGODB_URI` is defined then it replaces `uriF` and its default value.

Reply:
If a user starts `mongodb_exporter` with explicit `-mongodb.uri` flag in an environment that has `MONGODB_URI` defined (that may not be as explicit as the flag), we silently ignore the flag value. I think that is a problem. Maybe we could check that situation and exit with an error message, or at least log a warning.

Reply:
And/or we can make flag take priority over environment variable.

Reply:
LGTM otherwise. That one up to you.

Reply:
I know that. But, ah, the thing is it's not something I designed or changed here, just kept as is - lines moved from other method. If I change that it would be a breaking change.